src/integration_tests/test_provisioning.py

```diff
@@ -39,6 +40,13 @@ def test_create(self):
             lifecycle.start(stackname)
 
             cfn.cmd(stackname, "ls -l /", username=BOOTSTRAP_USER, concurrency='parallel')
+            cfn.download_file(stackname, "/bin/ls", "ls", use_bootstrap_user="true")
+            self.assertTrue(os.path.isfile("./ls"))
+            cfn.download_file(stackname, "/bin/less", "venv/bin/", use_bootstrap_user="true")
+            self.assertTrue(os.path.isfile("./venv/bin/less"))
+
+            cfn.download_file(stackname, "/bin/pwd", "subfolder/pwd", use_bootstrap_user="true")
```

Comment:
I ran the tests locally and can see `venv/bin/less` , `./ls` and `subfolder/pwd` are still hanging around. I'll tweak the code to remove them after running the test.